### DISCUSSION TITLE
[BUG] 运行这个jupyter报错，请问如何修改

### DISCUSSION THREAD

**Discussion by zhangj123** [alibaba/GraphScope#2259] [General]
**Describe the bug**
<img width="1141" alt="image" src="https://user-images.githubusercontent.com/44661598/204589757-ab5c9301-0a5b-4bde-9342-92cc10610009.png">

**Additional context**

**Comment by welcome** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template! And a maintainer will get back to you shortly!
Please feel free to contact us on [DingTalk](https://h5.dingtalk.com/circle/healthCheckin.html?dtaction=os&corpId=dingb4b76e0fa9ccd8d516fd0f0acdf19f55&68e71190=0fe34ec0&cbdbhh=qwertyuiop), WeChat account(graphscope) or [Slack](http://slack.graphscope.io/). We are happy to answer your questions responsively.

**Comment by sighingnow** [comment]
You need install java to use GIE.
